types/v1/types.go

```diff
@@ -122,6 +101,10 @@ type Result struct {
 	Plugin string `json:"plugin"`
 	// Version of the plugin
 	Version string `json:"version"`
+	// If the result is an error
+	IsError bool `json:iserror`
```

Comment:
we don't need field like this.  I would just have the `ErrorMessage` and then you can put a `.Err() error` method on this struct that returns `nil` if error message == "" or an errors.New(r.ErrorMessage)` if there is an error.  It makes the flow more clear

Reply:
done